apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -258,14 +258,14 @@ const AvailabilityPage = ({ profile, eventType, ...restProps }: Props) => {
                   weekStart={
                     typeof profile.weekStart === "string"
                       ? ([
-                          "Sunday",
-                          "Monday",
-                          "Tuesday",
-                          "Wednesday",
-                          "Thursday",
-                          "Friday",
-                          "Saturday",
-                        ].indexOf(profile.weekStart) as 0 | 1 | 2 | 3 | 4 | 5 | 6)
+                        "Sunday",
```

Comment:
Hey @kemilbeltre 
Could you please check the lint issues?

Reply:
@ujwalkumar1995 done. Thanks